src/ensembl/src/shared/types/help-and-docs/menu.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type Menu = {
+  name: string;
+  data: MenuItem[];
+};
+
+type MenuItem = MenuArticleItem | MenuCollectionItem;
+
+type MenuCollectionItem = {
+  name: string;
+  type: 'collection';
+  url?: string; // if a meenu directory has an index page associated with it
```

Comment:
`meenu` -> `menu`

Reply:
Damn this butterfly Macbook keyboard! 😠  It's keeps doing this to me, spontaneously duplicating the letter e and a couple of others whenever it feels like it.

Reply:
Haha! Looks like I am not alone with this :D